### ISSUE TITLE
Неупорядоченная выгрузка закладок

### ISSUE THREAD

**Issue by EvilBeaver** [TeamBIOS/cfg_tools#4]
Закладки выгружаются сначала по порядку, а потом порядок теряется. См. скриншоты:

![01-02-2016 14-17-50](https://cloud.githubusercontent.com/assets/2657561/12716059/f2cd5474-c8ee-11e5-8526-0bd0e1ebafe3.png)
![01-02-2016 14-17-55](https://cloud.githubusercontent.com/assets/2657561/12716058/f2cc2bee-c8ee-11e5-81dd-980b188e2bec.png)

Видно, что номера перепутаны. Причем на втором скриншоте перепутаны сильно.

Подозреваю, что таблица закладок читается из 1CD без применения индекса и ее надо принудительно сортировать.

**Comment by TeamBIOS** [comment]
Да, похоже придется реализовывать работу с индексом. На наших базах все по порядку было, поэтому не стали..

**Comment by EvilBeaver** [comment]
Там еще интереснее - см. второй скриншот. Там 112 версия несколько раз пытается выгружаться и коммититься. И подозрительно в тот же самый момент идут "Файл не принадлежащий объекту".

**Comment by TeamBIOS** [comment]
Будем пытаться отловить проблему. Без тестового примера, конечно, сложно будет воспроизвести...

**Comment by ghost** [comment]
@TeamBIOS добавьте просто формат логирования TRACE - так будет проще отлавливать всем

**Comment by TeamBIOS** [comment]
@allustin Можете по-подробнее, пожалуйста, про "просто формат логирования TRACE" желательно на python или ссылку, куда копать?

**Comment by artbear** [comment]
@TeamBIOS 

> формат логирования TRACE

``` python
logging.basicConfig(level=logging.INFO)  # DEBUG => print ALL msgs
```

вместо `INFO` или `DEBUG` можно поставить TRACE

ну и гуглить logging.basicConfig

**Comment by EvilBeaver** [comment]
@artbear и чего получится, если TRACE указать?

**Comment by TeamBIOS** [comment]
@artbear Нет уровня логирования TRACE
https://docs.python.org/3/howto/logging.html

**Comment by artbear** [comment]
@TeamBIOS @EvilBeaver не знаю, я просто разместил объяву :)

ИМХО по опыту `log4j` использование debug, как правило, достаточно на 99%

**Comment by TeamBIOS** [comment]
В файле настройки можно указать уровень логирования DEBUG https://github.com/TeamBIOS/cfg_tools#Файл-настройки

**Comment by ghost** [comment]
это немного из другой оперы - из трассировки программ

https://docs.python.org/2/library/trace.html
и 
http://www.dalkescientific.com/writings/diary/archive/2005/04/20/tracing_python_code.html
и
https://docs.python.org/2/library/pdb.html

используется когда уровня DEBUG недостаточно из стандартного логера
